framework/aster-frame/src/arch/x86/cpu.rs

```diff
@@ -212,6 +216,24 @@ impl TdxTrapFrame for GeneralRegs {
     }
 }
 
+impl UserPreemption {
+    const DEFAULT_PREEMPTION_COUNT: usize = 100;
```

Comment:
I wouldn't say this variable is a _default_ value. It is more like a period or interval.

In addition, even integer types with same value ranges (like `u64` and `usize`) have subtle semantic differences and should not be used interchangeably. The `usize` type has the pointer size, which means it is always large enough to address the entire memory space. So `usize` is used to represent memory addresses and slice indexes. The `u64` type represents a 64-bit number, so it is the best fit for counting. But `u64` here is unnecessarily long. So I recommend using `u32` or other smaller integer types.

```rust
const PREEMPTION_INTERVAL: u32 = 100;
```